sql/branch.py

```diff
@@ -0,0 +1,16 @@
+from gratipay import wireup
+
+db = wireup.db(wireup.env())
+
+participants = db.all("""
+    SELECT p.*::participants
+      FROM participants p
+      JOIN payment_instructions pi ON pi.participant = p.username -- Only include those with tips
+""")
+total = len(participants)
```

Comment:
Yes, and I think that's fine - the operations are idempotent so crash/rerun shouldn't be a problem.